src/common/utils/array.ts

```diff
@@ -22,4 +20,4 @@ export const findLastIndex = <T>(
 }
 
 export const asArray = <T>(item: T | undefined): T[] | undefined =>
-  isDefined(item) ? [item] : undefined
+  item != undefined ? [item] : undefined
```

Comment:
```suggestion
  item !== undefined ? [item] : undefined
```

Reply:
unless you were going for a check on both null and undefined, in which case let's change the signature to

`export const asArray = <T>(item: T | null | undefined): T[] | undefined`